composer/loggers/remote_uploader_downloader.py

```diff
@@ -671,6 +671,10 @@ def _upload_worker(
                 continue
         uri = remote_backend.get_uri(remote_file_name)
 
+        local_rank = dist.get_local_rank()
```

Comment:
Please add a comment to describe why we are doing this/ why someone may want to set this

Reply:
done